frontend/src/services/logger.js

```diff
@@ -0,0 +1,13 @@
+import winston from "winston";
+
+const logger = new winston.Logger({
```

Comment:
When starting up the app, I run into this error:

```
Error: winston.Logger was moved in winston@3.0.0.
Use a winston.createLogger instead.
```

After changing to `winston.createLogger` it works. Can you check to see if that change works for you too?

Reply:
updated in https://github.com/gambinish/stocks/pull/19/commits/4631c8f80e5ed8502207f2271da75c82ef69ea0e